proposed/http-client/http-client.md

```diff
@@ -27,8 +28,8 @@ MUST throw a `Psr\Http\Client\Exception\RequestException`. If there is an error
 with the network or the remote server cannot be reached, the HTTP client MUST throw
 a `Psr\Http\Client\Exception\NetworkException`.
 
-Smaller issues, like wrong HTTP versions, that are not blocking the HTTP client from
-sending the request MUST not cause any exception.
+Smaller issues that do not block the client from sending the request (such as
+invalid HTTP versions) MUST NOT result in exceptions.
```

Comment:
The example is a throw-away, so I made it parenthetical. Also, "MUST not" became "MUST NOT".